specification/keyvault/resource-manager/readme.md

```diff
@@ -218,11 +218,24 @@ input-file:
 directive:
 - suppress:
     - R3016 # The 'release_policy' property for KeyCreateParameters does not support camelCase.
+- suppress:
     - R3026 # The 'PrivateEndpointConnection' and 'PrivateLinkResource' sub-resources don't define PATCH as per Network Team's specification.
+- suppress:
     - R3025 # The 'PrivateLinkResource' is only accessible via List operation; does not define GET as per Network Team's specification.
+- suppress:
     - R4015 # The 'MHSMPrivateEndpointConnection' sub-resource doesn't define List as per Network Team's specification.
+- suppress:
     - R2005 # The 'ManagedHsms_PurgeDeleted' operation should not return a mix of 202 and syncronous return types (200, 201, 204) as directed by SDK team.
+- suppress:
     - R4009 # Vault object is the only one that need to be tracked with SystemData
+- suppress: INVALID_REQUEST_PARAMETER
+  from: keyvault.json
+  reason: The Vaults_List API endpoint only supports version 2015-11-01.
+- suppress: OBJECT_MISSING_REQUIRED_PROPERTY
+  from: keyvault.json
+  where:
```

Comment:
Can you remove `where` for now as this api-version error isn't supported by where suppression?

Reply:
Removed the where here: https://github.com/Azure/azure-rest-api-specs/pull/22155/commits/eb0e90c24fe3ba5f8b29c94a0a5a407a8797c504. The validation is failing: https://github.com/Azure/azure-rest-api-specs/pull/22155/checks?check_run_id=10665304132